SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/EditorWorkerController.h

```diff
@@ -45,20 +46,26 @@ struct EditorWorkerController
 		FSpatialGDKEditorToolbarModule& Toolbar = FModuleManager::GetModuleChecked<FSpatialGDKEditorToolbarModule>("SpatialGDKEditorToolbar");
 		SpatialShutdownHandle = Toolbar.OnSpatialShutdown.AddRaw(this, &EditorWorkerController::OnSpatialShutdown);
 
-		int32 PlayNumberOfServers;
-		GetDefault<ULevelEditorPlaySettings>()->GetPlayNumberOfServers(PlayNumberOfServers);
+		const ULevelEditorPlaySettings* LevelEditorPlaySettings = GetDefault<ULevelEditorPlaySettings>();
+		const int32 WorkerCount = LevelEditorPlaySettings->GetTotalServerWorkerCount();
```

Comment:
@tenevdev introduced this function, let me know if it will do what you need for your debugging settings?

Reply:
Yes, I'll use it but left a comment on the engine PR to clarify that this is not the count of connected servers, just the setting of how many to launch from editor.